app/operations/story_operations.rb

```diff
@@ -48,45 +47,86 @@ def notify_changes(story)
     end
 
     def create_activity(story, current_user)
```

Comment:
Same thing here. Why this method is public if you calling like this `StoryOperations::Create.new.call(story: @story, current_user: current_user)` ?

Reply:
can you explain better what you mean? I don't get it :smile: